Adding spatialTIME instructions (#3)
fhdsl/Moffitt_ITN_Workshop: sc-proteomics.Rmd

```{r echo = FALSE, show = FALSE}
knitr::opts_chunk$set(number_sections = FALSE)
```

> ## Activity: Set Posit Cloud

Reviewer: ```suggestion
## Activity: Set Up Posit Cloud
```

For easy start up we will use Posit Cloud -- an online resource for using R.

> <input type="checkbox"> If you have not already accept the invite to our Posit cloud space go to your email that you registered with and accept that invite. <br>  

Reviewer: ```suggestion
<input type="checkbox"> If you have not already, accept the invite to our Posit cloud space that was sent to your email (that you registered for this workshop with). <br>  
```

> The email will looks like this: 
> > 
> ______ has invited you to join Posit Cloud. By clicking on the link below, you can join the space that has been shared with you. Note that you will first be prompted to create an Posit Cloud account.<br> 

Reviewer: ```suggestion
The email will looks like this: <br/>
> 
______ has invited you to join Posit Cloud. By clicking on the link below, you can join the space that has been shared with you. Note that you will first be prompted to create an Posit Cloud account.<br> 
```

<input type="checkbox"> Create an account if needed. <br>  
<input type="checkbox"> Log into Posit.Cloud if needed <br>
<input type="checkbox"> Get a copy of this workspace https://posit.cloud/spaces/604196/content/9588231 You can also find it in the "Workshops" list <br>
<input type="checkbox"> Name your copy of your workspace by typing in on the title at the top.  <br>

![Posit Set Up Steps](./resources/images/posit_setup.png)

> <input type="checkbox">  On one of the early steps in the code it will ask you `.cache/R/ExperimentHub does not exist, creat directory? (yes/no)`. Type in "yes" and hit Enter. You should only have to do this once. <br>

Reviewer: ```suggestion
<input type="checkbox">  One of the early steps in the code will ask you `.cache/R/ExperimentHub does not exist, create directory? (yes/no)`. Type in "yes" and hit Enter. You should only have to do this once. <br>
```

> <input type="checkbox"> Now follow along with each step in the notebook and clicking the green triangles at each code chunk as we move along.

Reviewer: ```suggestion
<input type="checkbox"> Now follow along with each step in the notebook by clicking the green triangles at the top right of each code chunk.
```

**It is very important you run each step in order for the code to work!**

<input type="checkbox"> For this notebook, as you save the Rmd, the output is saved as a snapshot in the nb.html file. Click on this nb.html file and choose "View in web browser" <br>

## Bonus: To run on your own computer

> We will keep the Posit Cloud account available to you for one week after the workshop. Please download your files you'd like to keep before this week is up!

Reviewer: ```suggestion
We will keep the Posit Cloud account available to you for one week after the workshop. Please download any of your files you'd like to keep before this week is up!
```
